ext/openssl/openssl.c

```diff
@@ -3836,6 +3836,9 @@ static EVP_PKEY * php_openssl_evp_from_zval(
 			}
 
 			if (in == NULL) {
+				if (filename) {
+					php_error_docref(NULL, E_WARNING, "error opening file %s", filename);
```

Comment:
I think "warning : %s file not found." maybe better than "error opening file %s"